_posts/2019-03-13-activeadmin-vs-administrate.md

```diff
@@ -0,0 +1,41 @@
+
+A sensible alternative to this potential mess is to separate your administrative functions.  This can be done to either a specific area of the existing application, or even better yet, to a stand alone application.  In addition to isolating sensitive code to a completely separate code base, you can take additional security measures not related to the code at all, such as placing the admin functions inside a VPN.
+
+# Enter our contenders
+
+Since they're both implemented as Rails engines, to a large extent using either Administrate or ActiveAdmin will make the choice for you - you'll start out leaning heavily towards the separate application camp.  I make a note of that because, if you really want to just create an administrative overlay on your existing application, then neither one of these options will help you much.
+
+# ActiveAdmin
+
+If you're a big fan of DSLs, then take a look at  [ActiveAdmin](https://activeadmin.info).  The core pattern behind it is a straightforward DSL that lets you register a resource, and then add it's fields to a largely pre-defined interface.
+
+Between the two options, ActiveAdmin generally takes less work to stand up a running application, and provides much better filtering and sorting out of the box.  This is also a personal opinion, but the UI seems more polished.  However, I've found customizing the interface with additional javascript and UI flair like inline editing and dropdown menus to be more difficult.  That said, it's definitely a solid option, and one that you should lean towards if you just want to put up an admin app and things like filtering and sorting are really important for your use case.
+
+# Administrate
+
+If you prefer to stick to plain old Ruby and use Rails conventions, then [Administrate](https://administrate-prototype.herokuapp.com/getting_started) may be what you're looking for.  In keeping with general Rails philosophy, it adheres to a set of conventions to provide a pretty standard, "pre-canned" administrative application with minimal configuration needed.  However, if you need to extend or alter the standard functionality, then you can override the gem provided controllers, views, and styling elements using generators provided by the gem.  Just generate whatever you need to customize (views, controller actions), either for the app as a whole or for a specific resource, and edit away.  This makes Administrate more straightforward to customize if you have some more specific requirements.  This is good in general, and also (another personal opinion here) helps to make up for a rougher and less fully featured default interface.
```

Comment:
@rdormer I think one of the main advantages of `administrate` is that you don't *need to learn a new DSL* and in the long run it seems to be easier to tweak. Just my 2 cents. 

Also, no need to add so much of "this is my personal opinion" because this article is mostly an opinion on two gems.